charts/vmclarity/templates/apiserver/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-apiserver" (include "vmclarity.names.fullname" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "vmclarity.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: apiserver
+spec:
+  selector:
+    matchLabels: {{- include "vmclarity.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: apiserver
+  replicas: {{ .Values.apiserver.replicaCount }}
+  template:
+    metadata:
+      labels: {{- include "vmclarity.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: apiserver
+    spec:
+      containers:
```

Comment:
can you please create a service account for each component and use it in `serviceAccountName`?

Reply:
vmclarity currently has no need to access kubernetes resources, I can make the serviceAccountName configurable but I don't think we need to create a service account per component. The default service account for the namespace is the most locked down service account so that will give us the most security.

Reply:
To me it's more clear if every component will have it's own service account created, even if no permissions are needed. I don't like to use the `default` one.
Also, since currently we don't need to access and k8s resource we should set `automountServiceAccountToken: false` on the SA and/or pod so it will be more secure.

Reply:
added, please see the latest commits